{"uuid":"8599c71e-ba04-4453-8aa9-f2ad72e7f503","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["0eb9bcca-aa45-4e51-aaf0-d2bca4c9d6f8"],"befores":[],"afters":[{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608651843,"stop":1681608651845},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608652182,"stop":1681608652183},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608652301,"stop":1681608652301},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608652379,"stop":1681608652379},{"name":"hooks.Hook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1681608652416,"stop":1681608652417}],"start":1681608651640,"stop":1681608652418}